releasenotes/notes/remove-extensions-ce520ba419c93c58.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    Remove the ``qiskit.extensions`` module, which has been pending deprecation since the 0.45 release
+    and has been fully deprecated in the 0.46 release.
+    Most objects have been moved to :mod:`qiskit.circuit.library`, see the release notes of
+    the 0.45 release for more details.
+  - |
+    Removed deprecated, duplicated :class:`.QuantumCircuit` methods, such as ``fredkin`` or ``toffoli``.
```

Comment:
We should list all the aliases that have been removed and what they were replaced with, so if people's code is broken, they can easily search the release notes - I don't think we can absolutely 100% rely on all the release notes for past releases being available on docs.quantum.ibm.com.  They should be, but they might be split on separate pages, etc.